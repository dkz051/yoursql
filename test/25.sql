CREATE DATABASE oop;
USE oop;
CREATE TABLE qwe(id INT, stu INT, calculus INT, algebra INT, oop INT, PRIMARY KEY(id));
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 1, 1001, NULL,  100, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 2, 1002,   97,   87,   18);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 3, 1003,   89,   19,   81);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 4, 1004,   34, NULL,   56);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 5, 1005,   89,    3, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 6, 1006, NULL,   67,   90);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 7, 1007,   87,   46, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 8, 1008, NULL,   56,   52);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES( 9, 1009,   42, NULL,   74);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(10, 1010,   57,   57,   63);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(11, 1011,   34, NULL,   45);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(12, 1012, NULL,   84,   81);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(13, 1013,   74,   45, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(14, 1014,   68,   59,   35);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(15, 1015,    9,   76, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(16, 1016, NULL,   59,   25);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(17, 1017,   70,   65,   76);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(18, 1018,   90, NULL,   87);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(19, 1019,   76,   99,   45);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(20, 1020,   67,   60, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(21, 1021, NULL,  100,   34);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(22, 1022,    0,   76,   71);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(23, 1023,    5,   87, NULL);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(24, 1024,   76,   65,   37);
INSERT INTO qwe(id, stu, calculus, algebra, oop) VALUES(25, 1025,   57,   45,   98);
SELECT COUNT(calculus), AVG(calculus), MIN(calculus), MAX(calculus), STDDEV(calculus) FROM qwe;
SELECT COUNT(algebra), AVG(algebra), MIN(algebra), MAX(algebra), STDDEV(algebra) FROM qwe;
SELECT COUNT(oop), AVG(oop), MIN(oop), MAX(oop), STDDEV(oop) FROM qwe;
SELECT COUNT(*) FROM qwe;
SELECT SUM(calculus), SUM(algebra), SUM(oop) FROM qwe;
DROP DATABASE oop;
