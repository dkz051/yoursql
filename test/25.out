COUNT(calculus)	AVG(calculus)	MIN(calculus)	MAX(calculus)	STDDEV(calculus)
19	59.0000	0	97	29.470770532673107
COUNT(algebra)	AVG(algebra)	MIN(algebra)	MAX(algebra)	STDDEV(algebra)
21	64.5238	3	100	24.435764280327287
COUNT(oop)	AVG(oop)	MIN(oop)	MAX(oop)	STDDEV(oop)
18	59.3333	18	98	23.426006441085466
COUNT(*)
25
SUM(calculus)	SUM(algebra)	SUM(oop)
1121	1355	1068
